### ISSUE TITLE
[Feature Request] Have 10x executable be able to find project without specifying .10x extension

### ISSUE THREAD

**Issue by lucrool** [slynch8/10x#2707]
![image](https://github.com/user-attachments/assets/9baf6c7c-4f86-4e1c-b35d-963b36048e7f)

10x would search for a .10x file within the directory its launched

**Comment by slynch8** [comment]
Sure. 

Btw, 10x already initialises the vcvarsall.bat env vars, see the VisualStudioEnvBat setting

**Comment by lucrool** [comment]
It doesn't automatically do that for me. Am I doing something wrong?

https://github.com/user-attachments/assets/cab24828-f295-4d49-8bc9-3c35a9c53609

![image](https://github.com/user-attachments/assets/62907e13-077e-4a9c-853a-bd025737dd75)
![image](https://github.com/user-attachments/assets/c207735e-796a-4630-972a-e1f3e45c71e0)

**Comment by slynch8** [comment]
For 10x workspaces you need to check the "Use Visual Studio Build Environment" checkbox in the workspace settings

![image](https://github.com/user-attachments/assets/e15bd2b3-0f28-4bf5-81b2-4e220d581cc2)

**Comment by lucrool** [comment]
Very Sweet. Very nice.
But, setting up 10x to use window's development environment wasn't straight forward. The docs don't give directions on how to set it up, I struggled this far forward by what someone told me to do and watching the only other resource I could find on the 10x (a YouTube video).
Though it's obvious you're supposed to check "Use Visual Studio Build Environment" having to enable it in two places is confusing (especially with there being no guide).

> # Path of the Visual Studio VsDevCmd.bat or vcvars64.bat batch file to use for setting the Visual Studio build environment. Restart 10X after changing this value. Set to 'auto' to let 10x try and find the path
Doesn't mention the checkbox (But wow, it's so obvious now that I needed to enable it).

Don't want to sound like I'm complaining, just want to help improve the editor so I'm being maybe pickier than I need to be. I like 10x a lot.
